apps/accounts/templates/provider_portal/provider_domain_new/base.html

```diff
@@ -24,8 +24,17 @@ <h2 class="mt-4">
     <article>
       <div class="prose mx-auto mt-8">
         {% block explanatory_copy %}
-            <p class="italic">Explanatory copy here about how Carbon.txt gives you more control over how your provider is presented on Green Web Check results and Green Web  badges. </p>
-            <p>More copy explaining what is required (access to server, DNS, etc), and how long the process will take and what it will involve</p>
+            <p>Carbon.txt is a single recognisable location on any web domain for public sustainability data relating to that company. By combing Carbon.txt with your Green Web Foundation provider listing you can maintain a single source of truth for your organisations sustainability disclosures, and then share them across the different domains your organisation owns.</p>
+            <p>This benefit can also be passed on to your customers, who's domains can also be linked to your hosting provider listing our dataset. This ensures that whenever their domains are checked in our <a href="https://www.thegreenwebfoundation.org/green-web-check/" target="_blank">Green Web Check</a> and <a href="https://admin.thegreenwebfoundation.org/api-docs/" target="_blank">API</a> the results accurately reflect their use of your services.</p>
+
+            <p>To continue, you will need either:</p>
```

Comment:
Just one tiny thing here, i think we need an `{% if provider.primary_linked_domain %}` conditional here, which displays the text as written *IF* a primary domain is set, and just "The ability to upload a Carbon.txt file to your website server." otherwise? The validator won't complain if you point your primary domain at a domain with A DNS TXT record or HTTP header instead of an actual self-hosted carbon.txt, but it's not really a real use case, and we maybe want to be a bit more explicit about that

Reply:
I was thinking about this as well. I did have that gate in there previously, but removed it.
It does make sense that one primary domain _should_ always have its own carbon.txt file though.

For this to be complete, we would:

1. Need to put a gate here to check if the primary domain already exists.
2. Need to update the previous step (where the user inputs the domain). That step should:
    1. Check if a primary domain already exists for the provider.
    2. **If so** - show all the options as they currently are.
    3. **If not** - hide the option for the user to select the kind of domain they are linking & set the input domain as their primary domain by default (probably a hidden form field on the page).